CenterForOpenScience/osf.io
# [ENG-54] Send user to mailchimp sign-up form for re-subscription

## Purpose

Mailchimp has outlawed resubscribing users via their API so in order to allow users to resubscribe need to redirect users to their external form and then update our records.

## Changes

![email-fix](https://user-images.githubusercontent.com/9688518/156784762-b5294ded-20f3-49ed-987a-b4de40d5b2c7.png)

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify you can unsubscribe and re-subscribe to the general osf mailing list

## Documentation

🐞  fix, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-54

Legacy:
https://openscience.atlassian.net/browse/ENG-57
